src/Stryker.Core/Stryker.Core.UnitTest/InjectedHelpers/InjectedHelperTests.cs

```diff
@@ -27,8 +28,15 @@ public void InjectHelpers_ShouldCompile()
                 }
             }
 
+            var syntaxes = new List<SyntaxTree>();
+
+            foreach (var helper in CodeInjection.MutantHelpers)
```

Comment:
I think we should delete this test instead of keeping it up to date. The theory below also tests the same for langversion Latest